easybuild/easyconfigs/c/CGmapTools/CGmapTools-0.1.2_add-Makefile.patch

```diff
@@ -0,0 +1,29 @@
+Create custom Makefile
+
+author: Pavel Grochal (INUITS)
+--- Makefile.orig	2020-03-30 17:12:39.099451802 +0200
++++ Makefile	2020-03-30 17:59:38.570459472 +0200
+@@ -0,0 +1,23 @@
++CC ?= gcc
++CXX ?= g++
++CFLAGS ?= -g
++CXXFLAGS ?= -g
```

Comment:
Please also add `-O2` here:

```makefile
CFLAGS ?= -g -O2
CXXFLAGS ?= -g -O2
```

Compiling without any `-Ox` flag is equivalent with using `-O0` with `gcc`/`g++`, which implies no optimizations at all (and hence terribly slow binaries), which should be avoided.

Don't forget to update the SHA256 checksum for this patch file.